src/main/java/Task.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * The Task class provides a template for the Task object where description and status is stored
+ */
+public class Task {
+
+    public static final String TICK_ICON = "\u2713";
+    public static final String CROSS_ICON = "\u2718";
+
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns tick or X icons
+     */
+    public String getStatusIcon() {
+        return (isDone ? TICK_ICON : CROSS_ICON);
+    }
+
+    /**
+     * Sets isDone to true
+     */
```

Comment:
Perhaps some of the comments such as these could be removed, they are quite self documenting.

Reply:
Thank you Abner for pointing that out, I will update the use of comments